files/initramfs-tools/union-mount.j2

```diff
@@ -11,6 +11,31 @@ case $1 in
     ;;
 esac
 
+set_tmpfs_log_partition_size()
+{
+  varlogsize=128
+
+  # NOTE: certain platforms, when reaching initramfs stage, have a small
+  #       limit of mounting tmpfs partition, potentially due to amount
+  #       of RAM available in this stage. e.g. Arista 7050 QX32
+  [ X"$aboot_platform" = X"x86_64-arista_7050_qx32"  ] && return
```

Comment:
Can we extend this to the 7060-CX32 as well?

Reply:
Return here would cause small /var/log partition of 128M, are you sure you want 7060 here?

Reply:
128M is already large IMHO so OK for me

Reply:
:-)
I looked at a device. I think we currently mount /var/log for 380M in loop file system. With the 5% to 10% adjustment will mount tmpfs of 512M. That doesn't sound too bad to me.